factory-wireguard.py

```diff
@@ -200,7 +201,9 @@ def gen_conf(self, factory: str) -> str:
         self._gen_conf(factory, buf)
         return buf.getvalue()
 
-    def apply_conf(self, factory: str, conf: str, intf_name: str):
+    def apply_conf(self, factory: str, conf: str, intf_name: str, no_sysctl: bool):
+        if no_sysctl:
+            conf = re.sub(".*sysctl.*\n?","", conf)
```

Comment:
how about we pass `no_sysctl` to the `gen_conf` method in a way that we won't have to do regex fixes of this file